requirements.txt

```diff
@@ -1,7 +0,0 @@
-iniconfig==2.1.0
-packaging==25.0
-pluggy==1.6.0
-Pygments==2.19.1
-pytest==8.4.0
```

Comment:
You've lost the pytest dev dependency, which belongs in a dependency group.

Reply:
I did not see any tests, so I removed it.

Reply:
I'm sure @hardwarehaven is planning to add some :P

Reply:
I think so, but until then it's not needed.
It's just `uv add pytest --group dev` anyway.